integration-cli/docker_cli_kill_test.go

```diff
@@ -63,12 +68,14 @@ func (s *DockerSuite) TestKillWithSignal(c *check.C) {
 func (s *DockerSuite) TestKillWithStopSignalWithSameSignalShouldDisableRestartPolicy(c *check.C) {
 	// Cannot port to Windows - does not support signals int the same way as Linux does
 	testRequires(c, DaemonIsLinux)
-	out, _ := dockerCmd(c, "run", "-d", "--stop-signal=TERM", "busybox", "top")
+	out, _ := dockerCmd(c, "run", "-d", "--stop-signal=TERM", "--restart=always", "busybox", "top")
 	cid := strings.TrimSpace(out)
 	c.Assert(waitRun(cid), check.IsNil)
 
-	// Let's docker send a CONT signal to the container
+	// Let docker send a TERM signal to the container
+	// It will kill the process and disable the restart policy
 	dockerCmd(c, "kill", "-s", "TERM", cid)
+	c.Assert(waitExited(cid, 10*time.Second), check.IsNil)
 
 	out, _ = dockerCmd(c, "ps", "-q")
```

Comment:
We could and thus we would use `checker.Equals` instead of `checker.Contains` in the assert below 👼